### PR TITLE
Dynamically add event listeners

### PR DESCRIPTION
**Pull request by josueJURE**
- I tried to center the Font Awesome paper-plane icon within its parent container, but that can’t be done due to its built-in `CSS `property that cannot be altered.

- I used object destructuring in the `server.js file`, which enhances readability by reducing the need for repetitive access to object properties.

- I’ve also dynamically added event listeners to the stop, pause, and play `buttons`. The speed functionality is still a work in progress.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -50,28 +50,26 @@ app.get("/email", async (req, res) => {
 
 
 app.get("/openai", async (req, res) => {
-  console.log(req.query);
+  
+  const {recipe_country_of_origin, is_lactose_intolerant, is_vegan, what_are_user_other_dietary_requirements } = req.query
```
Comment (samscupoftea) [inline server.js:54]: I see what you've done here. It's a noticeable improvement! Great work @josueJURE

File: public/index.js [hunk 2]
```diff
@@ -282,11 +286,18 @@ recipeButtons.forEach((button) => {
...
+            const speechBtns = Array.from(
+              document.querySelectorAll(".fa-solid")
+            );
+            const speedBtn = document.querySelector("#speed");
+
+            console.log(speechBtns);
+
             function readRecipe(recipe) {
-              if(speechSynthesis.paused && speechSynthesis.speaking) {
+              if (speechSynthesis.paused && speechSynthesis.speaking) {
```
Comment (samscupoftea) [inline public/index.js:297]: This is interesting. I've never encountered ".paused" or ".speaking" in Javascript. I didn't realise they existed! @josueJURE

**Review by samscupoftea — COMMENTED** [review]
Great work @josueJURE 
I am creating a Readme along with a video to accompany the project.

File: public/index.js (hunk 2 above)
Comment (josueJURE) [reply public/index.js:297]: have a look at this link

https://developer.mozilla.org/en-US/docs/Web/API/SpeechSynthesis

**Pull request merged**